### PR TITLE
MTM-58246 Update the Web SDK with information about Angular 17

### PR DESCRIPTION
**Pull request by DawidSAG**
Angular 17 is merged to the develop

### CODE REVIEW

File: content/web/gettingstarted.md [hunk 1]
```diff
@@ -9,8 +9,15 @@ section:
...
+Use `latest` tag if the current version available in the NPM is 17
 ```bash
-npx @angular/cli@v16-lts new --style=less
+npx @angular/cli@latest new --style=less --standalone=false
+```
+
+or
+
+```bash
+npx @angular/cli@v17-lts new --style=less --standalone=false
```
Comment (reey) [inline content/web/gettingstarted.md:20]: Why not just `v17-lts` ?

File: content/web/overview.md [hunk 2]
```diff
@@ -100,6 +100,7 @@ As our releases are bound to the Angular versioning, you must to ensure that you
 
 | Angular version | Web SDK version | Comment |
 | --- | --- | --- |
+| 17.x.x | 2020.x.x | No support for the standalone flag |
```
Comment (reey) [inline content/web/overview.md:103]:
```suggestion
| 17.x.x | 1020.x.x | No support for the standalone flag |
```

**Review by reey — APPROVED** [review]

File: content/web/upgrade-bundle/upgrading-to-angular-17.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+---
+title: Upgrading from Angular 16 to Angular 17  
+layout: redirect
+weight: 10
+---
+
+Angular 17 is supported from version `1020.0.0`. The following configuration changes are required before you can run the application:
+
+- Update all `@c8y` dependencies to version `1020.x.x` in your `package.json`.
```
Comment (BeateRixen) [inline content/web/upgrade-bundle/upgrading-to-angular-17.md:10]:
```suggestion
- Update all `@c8y` dependencies to version `1020.x.x` in your *package.json*.
```

File: content/web/upgrade-bundle/upgrading-to-angular-17.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+- Replace `import 'zone.js/dist/zone'` with `import 'zone.js'` in the `src/polyfills.ts` file.
```
Comment (BeateRixen) [inline content/web/upgrade-bundle/upgrading-to-angular-17.md:11]:
```suggestion
- Replace `import 'zone.js/dist/zone'` with `import 'zone.js'` in the *src/polyfills.ts* file.
```

File: content/web/upgrade-bundle/upgrading-to-angular-17.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- Replace all occurrences of `"browserTarget"` with `"buildTarget"` in the `angular.json` file.
```
Comment (BeateRixen) [inline content/web/upgrade-bundle/upgrading-to-angular-17.md:12]:
```suggestion
- Replace all occurrences of `"browserTarget"` with `"buildTarget"` in the *angular.json* file.
```

File: content/web/gettingstarted.md (hunk 1 above)
Comment (DawidSAG) [reply content/web/gettingstarted.md:20]: My mistake `v17-lts` is not yet created on the npm. Version 17 is under the default/latest tag

**Review by sag-lukjan — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@DawidSAG This can probably be merged now.

**Pull request merged**